src/Experimental/Traversals/bfs.jl

```diff
@@ -0,0 +1,157 @@
+import Base.Sort, Base.Sort.Algorithm
+import Base:sort!
+
+struct NOOPSortAlg <: Base.Sort.Algorithm end
```

Comment:
@sbromberger this is not a subtype of `TraversalAlgorithm`, so the method `visited_vertices` is not applicable to it

Reply:
Nevermind, misread the question on Slack..